test/unit/core/front.test.ts

```diff
@@ -0,0 +1,476 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { expect } from 'chai';
+import { BigNumber } from 'ethers';
+import hre, { ethers } from 'hardhat';
+import { AttestationsRegistry, Front, MockAttester } from 'types';
+import { AttestationStruct, RequestStruct } from 'types/HydraS1SimpleAttester';
+import { evmRevert, evmSnapshot, increaseTime } from '../../../test/utils';
+
+type IssuerRange = {
+  min: number;
+  max: number;
+};
+
+type Requests = {
+  first: RequestStruct;
+  second: RequestStruct;
+};
+
+type Attestations = {
+  first: AttestationStruct;
+  second: AttestationStruct;
+};
+
+describe('Test Front contract', () => {
+  let userDestination: SignerWithAddress;
+  let secondUserDestination: SignerWithAddress;
+
+  let front: Front;
+  let attestationsRegistry: AttestationsRegistry;
+  let mockAttester: MockAttester;
+
+  let frontAuthorizedRange: IssuerRange;
+  let mockAttesterAuthorizedRange: IssuerRange;
+  let generationTimestamp: number;
+  let requests: Requests;
+  let attestations: Attestations;
+
+  before(async () => {
+    [userDestination, secondUserDestination] = await ethers.getSigners();
+
+    frontAuthorizedRange = {
+      min: 0,
+      max: 1,
+    };
+
+    mockAttesterAuthorizedRange = {
+      min: 2,
+      max: 3,
+    };
+
+    generationTimestamp = Math.floor(Date.now() / 1000);
+
+    requests = {
+      first: {
+        claims: [
+          {
+            groupId: 0,
+            claimedValue: 1,
+            extraData: ethers.utils.defaultAbiCoder.encode(['uint32'], [generationTimestamp]),
+          },
+        ],
+        destination: userDestination.address,
+      },
+      second: {
+        claims: [
+          {
+            groupId: 1,
+            claimedValue: 1,
+            extraData: ethers.utils.defaultAbiCoder.encode(['uint32'], [generationTimestamp]),
+          },
+        ],
+        destination: userDestination.address,
+      },
+    };
+  });
+
+  describe('Deployments', () => {
+    it('Should deploy, setup and test the constructed values of the contract', async () => {
+      ({
+        front,
+        mockAttester: mockAttester,
+        attestationsRegistry,
+      } = await hre.run('deploy-mock-attester-and-core', {
+        uri: 'https://token_cdn.domain/',
+        frontFirstCollectionId: frontAuthorizedRange.min.toString(),
+        frontLastCollectionId: frontAuthorizedRange.max.toString(),
+        collectionIdFirst: mockAttesterAuthorizedRange.min.toString(),
```

Comment:
maybe rename the variable to mockAttestationFirstCollectionId, or standardize the naming

Reply:
I'll standardize the name by replacing min & max by first & last